Bundle/CriteriaBundle/Resources/translations/victoire.fr.xliff

```diff
@@ -34,6 +34,25 @@
                 <source>victoire_criteria.criteria.operator.help_block</source>
                 <target>Valeurs prises en charge: equal, true, false, is_granted, is_not_granted</target>
             </trans-unit>
+            <trans-unit id="8" resname="victoire_criteria.criteria.operator.equal.label">
```

Comment:
Why did you add this translation ? It is not used in Victoire. I think you should remove it in all languages